synapse/axon.py

```diff
@@ -669,38 +826,64 @@ def find(self, htype, hvalu):
         '''
         Returns a list of blob tufos for hashes in the axon.
 
-        Example:
+        Args:
+            htype (str): Hash type.
+            hvalu (str): Hash value.
 
-            blobs = axon.find('sha256',x)
+        Examples:
+            Find all blobs for a given md5sum::
 
+                blobs = axon.find('md5', md5hash)
+
+        Returns:
+            list: List of tufos for a given hash value.
         '''
         return self.core.getTufosByProp('axon:blob:%s' % htype, valu=hvalu)
 
     def bytes(self, htype, hvalu):
         '''
         Yield chunks of bytes for the given hash value.
 
-        Example:
+        Args:
+            htype (str): Hash type.
+            hvalu (str): Hash value.
 
-            for byts in axon.bytes('md5',md5sum):
-                fd.write(byts)
+        Examples:
+            Write bytes for a md5 to disk::
 
+                for byts in axon.bytes('md5',md5sum):
+                    fd.write(byts)
+
+        Returns:
+            bytes:  A chunk of bytes
         '''
         if htype == 'guid':
             blob = self.core.getTufoByProp('axon:blob', valu=hvalu)
         else:
             blob = self.core.getTufoByProp('axon:blob:%s' % htype, valu=hvalu)
-        return self.iterblob(blob)
+        if blob:
```

Comment:
The previous behavior returned a generator; while the docs said it yielded bytes. This makes the behavior more explicit.